test/evergreen.yml

```diff
@@ -187,23 +187,23 @@ tasks:
       - func: "get project"
       - func: "compile wiredtiger"
         vars:
-          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ CFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb"
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb"
```

Comment:
I guess this change is to enable other toolchain binaries (e.g gcov) to be applied when executing the tests?

Reply:
Theoretically we should enforce all the tests using the consistent binaries to run even though they are not running the tests that are affected by mismatched binaries.

Reply:
Sounds good. Hopefully, there won't be breakage to other tests (or we are ready to fix if any).